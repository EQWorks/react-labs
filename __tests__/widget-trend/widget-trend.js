import React from 'react'
import renderer from 'react-test-renderer'
import WidgetTrend from '../../src/widget-trend';


describe('<WidgetTrend/>', () => {
  it('should render successfully', async () => {
    let data = {
      title: 'Untitled',
      value: 'N/A',
      percentage: 'N/A'
    }

    const component = renderer.create(
      <WidgetTrend {...data} />
    )

    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
