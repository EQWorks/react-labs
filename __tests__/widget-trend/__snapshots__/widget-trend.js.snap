// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WidgetTrend/> should render successfully 1`] = `
<div
  className="MuiPaper-root makeStyles-paper-1 MuiPaper-outlined MuiPaper-rounded"
>
  <h6
    className="MuiTypography-root makeStyles-title-2 MuiTypography-subtitle2 MuiTypography-gutterBottom"
  >
    Untitled
  </h6>
  <h5
    className="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom"
  >
    N/A
  </h5>
  <p
    className="MuiTypography-root MuiTypography-body1"
  >
    <svg
      aria-hidden="true"
      className="MuiSvgIcon-root"
      focusable="false"
      style={
        Object {
          "color": "green",
        }
      }
      viewBox="0 0 24 24"
    >
      <path
        d="M16.85 17.15l1.44-1.44-4.88-4.88-3.29 3.29c-.39.39-1.02.39-1.41 0l-6-6.01a.9959.9959 0 010-1.41c.39-.39 1.02-.39 1.41 0L9.41 12l3.29-3.29c.39-.39 1.02-.39 1.41 0l5.59 5.58 1.44-1.44c.31-.31.85-.09.85.35v4.29c0 .28-.22.5-.5.5H17.2c-.44.01-.66-.53-.35-.84z"
      />
    </svg>
    N/A
    % decrease from yesterday
  </p>
</div>
`;
